collections/_pages/index.md

```diff
@@ -57,30 +72,75 @@ Climate change is a complex of interrelated phenomena. That is why the data that
 </div> <!-- accordion end -->
 
 </div></div>
-
 <div class="section"><div class="container clearfix" markdown="1">
-{:#o-projektu .display-2}
-## About us and our work
+{:#about .display-2}
+## About our work
 
-{:.lead}
-We would like to contribute to a more evidence-based Czech public debate about climate change.
-We create easy-to-understand visual and text materials and communicate the latest research findings to support key decisions in the Czech Republic that are related to climate change and a transition to a net zero economy.
+<div class="row about-us lead mb-5 justify-content-between">
+<div class="col-12 col-md-6 pt-2 pt-md-4" markdown="1">
 
-{:.lead}
-The debate about climate change is complicated and full of emotions, half-truths and myths. We want to highlight the most important issues and build bridges between scientists and all people affected by climate change. Which means all people.
+**We do research and analytical work**. We browse the data and look for aspects relevant to a deeper understanding of climate change, its impacts and possible solutions. We accentuate the regional matters of Czechia and Europe but do not omit global views.
```

Comment:
alternative (not feeling strongly)

We sift through the data, searching for bits that help a deeper understanding of climate change, its impacts and possible solutions.